File: LeetCode/C/136-bit-manipulation-只出现一次的数字.cpp
/*
 * @lc app=leetcode.cn id=136 lang=cpp
 *
 * [136] 只出现一次的数字
 */
#include "main.h"

// @lc code=start
class Solution {
  public:
    int singleNumber(vector<int> &nums) {
        int res = 0;
        for (int num : nums)
            res ^= num;
        return res;
    }
};
// @lc code=end

File: LeetCode/Go/136-bit-manipulation-只出现一次的数字.go
/*
 * @lc app=leetcode.cn id=136 lang=golang
 *
 * [136] 只出现一次的数字
 */

package leetcode

// @lc code=start
func singleNumber(nums []int) int {
	var res int
	for _, num := range nums {
		res ^= num
	}
	return res
}

// @lc code=end
